pkg/cleanup/built_in_types.go

```diff
@@ -0,0 +1,72 @@
+package cleanup
+
+import (
+	"fmt"
+
+	sc_v1a1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	unstructured_conversion "k8s.io/apimachinery/pkg/conversion/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	apps_v1b1 "k8s.io/client-go/pkg/apis/apps/v1beta1"
+	ext_v1b1 "k8s.io/client-go/pkg/apis/extensions/v1beta1"
+)
+
+var converter = unstructured_conversion.NewConverter(false)
+
+var MainKnownTypes = map[schema.GroupKind]Cleanup{
+	{Group: apps_v1b1.GroupName, Kind: "Deployment"}: deploymentAppsCleanup,
+	{Group: ext_v1b1.GroupName, Kind: "Deployment"}:  deploymentExtCleanup,
+}
+
+var ServiceCatalogKnownTypes = map[schema.GroupKind]Cleanup{
```

Comment:
I was thinking maybe move these maps from here and ready checker to the app package? This code should not depend on all the types, it is generic? I'm not sure...